### ISSUE TITLE
"Configure Excludes" appearing after latest version update

### ISSUE THREAD

**Issue by imeredith** [microsoft/vscode#105456]
- VSCode Version: 1.48,2
- OS Version: Windows 10 Build 19041, remote ssh to local ubuntu 19.04 hyper-v vm

Steps to Reproduce:

Open editor on this current project (which is not public unforunatly

I have set logging to verbose and...

```
Info 415  [10:37:18.55] Before ensureProjectForOpenFiles:
Info 416  [10:37:18.55] Project '<projectRoot>/react/jsconfig.json' (Configured)
Info 416  [10:37:18.55] 	Files (393)

Info 416  [10:37:18.55] -----------------------------------------------
Info 416  [10:37:18.55] Project '/dev/null/inferredProject1*' (Inferred)
Info 416  [10:37:18.55] 	Files (86)
```

My project structure is

```
<root> 
   - react
       jsconfig.json
   - php
``` 

jsconfig.json 
```
{
    "compilerOptions": {
        "baseUrl": ".",
        "paths": {
            "@/*": ["./src/*"]
        }
    },
    "exclude": ["node_modules"]
}
```

I'm kind of at a loss as to why this has just started happening. I doubt it has anything to do with version due to a lack of other reports but timing was weird

Any more ideas? I'm not sure if there are other parts of server log that might be relevant 

Does this issue occur when all extensions are disabled?: N/A - I need extensions to even use this

**Comment by imeredith** [comment]
Also, I was on a version less than 1.48 before upgrade

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled) (leave the built-in JS/TS version enabled)?

The insiders build includes a major new TS version so I want to see if this fixes it

**Comment by imeredith** [comment]
So... yes it did work with insiders but then I tried stable again, and it worked. Very confused.

The only thing that was different about what I tried was that I was using "Restart TS language server" to test changes, and not restarting VSCode. I guess if the instance of the remote-ssh service was the issue then that could be it.

Thanks for help, will close this issue. If I see it again I will try  "restart language server" -> reconnect to remote ssh -> restart vscode
